dfttk/workflows.py

```diff
@@ -1001,6 +1167,7 @@ def kpoints_conv_test(
     ],
     force_gamma: bool = True,
     backup: bool = False,
+    max_errors: int = 10,
```

Comment:
why change this to 10?

Reply:
For some of my calculations, 3 is not enough.